tests/udf_framework/test_create_event_handler_udf_mock.py

```diff
@@ -0,0 +1,143 @@
+from pathlib import PurePosixPath
+from tempfile import TemporaryDirectory
+
+from exasol_bucketfs_utils_python.bucketfs_factory import BucketFSFactory
+from exasol_udf_mock_python.column import Column
+from exasol_udf_mock_python.connection import Connection
+from exasol_udf_mock_python.group import Group
+from exasol_udf_mock_python.mock_exa_environment import MockExaEnvironment
+from exasol_udf_mock_python.mock_meta_data import MockMetaData
+from exasol_udf_mock_python.udf_mock_executor import UDFMockExecutor
+from tests.udf_framework import mock_event_handlers
+
+
+def _udf_wrapper():
+    from exasol_udf_mock_python.udf_context import UDFContext
+    from exasol_advanced_analytics_framework.udf_framework.\
+        create_event_handler_udf import CreateEventHandlerUDF
+
+    udf = CreateEventHandlerUDF(exa)
+    def run(ctx: UDFContext):
+        udf.run(ctx)
+
+
+def _create_mock_data():
+    meta = MockMetaData(
+        script_code_wrapper_function=_udf_wrapper,
+        input_type="SET",
+        input_columns=[
+            Column("1", int, "INTEGER"),  # iter_num
+            Column("2", str, "VARCHAR(2000000)"),  # bucketfs_connection_name
+            Column("3", str, "VARCHAR(2000000)"),  # event_handler_class_name
+            Column("4", str, "VARCHAR(2000000)"),  # event_handler_module
+            Column("5", str, "VARCHAR(2000000)"),  # event_handler_parameters
+         ],
+        output_type="EMITS",
+        output_columns=[
+            Column("outputs", str, "VARCHAR(2000000)")
+        ],
+        is_variadic_input=True
+    )
+    return meta
+
+
+def test_event_handler_udf_with_one_iteration():
+    executor = UDFMockExecutor()
+    meta = _create_mock_data()
+
+    with TemporaryDirectory() as path:
+        bucketfs_connection = Connection(address=f"file://{path}/event_handler")
+        exa = MockExaEnvironment(
+            metadata=meta,
+            connections={"bucketfs_connection": bucketfs_connection})
+
+        input_data = (
+            0,
+            "bucketfs_connection",
+            "MockEventHandlerWithOneIteration",
+            "tests.udf_framework.mock_event_handlers",
+            "{}"
+        )
+        result = executor.run([Group([input_data])], exa)
+        for i, group in enumerate(result):
+            result_row = group.rows
+            assert len(result_row) == 4
```

Comment:
combine asserts

Reply:
why do you want to combine them? I did it but I didnt get the point

Reply:
Pytest doesn't really work with multiple asserts. The first assert which hits throws an exception and everything else is not evaluated. There are cases where this is fine, but in others not

  https://www.goshpat.com/multiple-assert-statements.html